docs/README.md

```diff
@@ -194,99 +194,197 @@ This method registers a callback function which is called once a node has loaded
-| _node_.studies_table_add(_params_,...) | `/studies/{studyDbId}/table` | `POST` |
-| _node_.studies_table(_params_,...) | `/studies/{studyDbId}/table` | `GET` |
-| _node_.studies(_params_,...) | `/studies` | `GET` |
-| _node_.studytypes(_params_,...) | `/studytypes`(>=v1.1) or `/studyTypes`(<v1.1) | `GET` |
-| _node_.traits_detail(_params_,...) | `/traits/{traitDbId}` | `GET` |
-| _node_.traits(_params_,...) | `/traits` | `GET` |
-| _node_.trials_detail(_params_,...) | `/trials/{trialDbId}` | `GET` |
-| _node_.trials(_params_,...) | `/trials` | `GET` |
-| _node_.variables_datats(_params_,...) | `/variables/datatypes` | `GET` |
-| _node_.variables_detail(_params_,...) | `/variables/{observationVariableDbId}` | `GET` |
-| _node_.variables_search(_params_,...) | `/variables-search` | `POST` |
-| _node_.variables(_params_,...) | `/variables` | `GET` |
-| _node_.vendor_orders_plates(_params_,...) | `/vendor/orders/{orderId}/plates` | `GET` |
-| _node_.vendor_orders_results(_params_,...) | `/vendor/orders/{orderId}/results` | `GET` |
-| _node_.vendor_orders_status(_params_,...) | `/vendor/orders/{orderId}/status` | `GET` |
-| _node_.vendor_orders(_params_,...) | `/vendor/orders` | `GET` |
-| _node_.vendor_plates_detail(_params_,...) | `/vendor/plates/{submissionId}` | `GET` |
-| _node_.vendor_plates_search(_params_,...) | `/vendor/plates-search`(>=v1.2) or `/vendor/plate-search`(<v1.2) | `POST` |
-| _node_.vendor_plates(_params_,...) | `/vendor/plates` | `POST` |
-| _node_.vendor_specifications(_params_,...) | `/vendor/specifications` | `GET` |
+| BrAPI.js Method | BrAPI Call (<= v1.3) | BrAPI Call (v2.0) | Default HTTPMethod |
+| --------------- | -------------------- | --------------- | ------------------ |
+| _node_.allelematrices_search(_params_,...) | `/allelematrices-search`(>=v1.2) or `/allelematrix-search`(<v1.2) || `POST` |
+| _node_.allelematrices(_params_,...) | `/allelematrices` | | `GET` |
+| _node_.attributes_categories(_params_,...) | `/attributes_categories` | `attributes/categories` | `GET` |
+| _node_.attributes_detail(_params_,...)||`/attributes/{attributeDbId}` | `GET` |
+| _node_.attributes_modify(_params_,...)||`/attributes/{attributeDbId}` | `PUT` |
+| _node_.attributes_store(_params_,...)||`/attributes` | `POST` |
+| _node_.attributes(_params_,...) | `/attributes` | `/attributes` | `GET` |
+| _node_.attributevalues_detail(_params_,...)||`/attributevalues/{attributeValueDbId}` | `GET` |
+| _node_.attributevalues_modify(_params_,...)||`/attributevalues/{attributeValueDbId}` | `PUT` |
+| _node_.attributevalues_store(_params_,...)||`/attributevalues` | `POST` |
+| _node_.attributevalues(_params_,...)||`/attributevalues` | `GET` |
+| _node_.breedingmethods_detail(_params_,...) | `/breedingmethods/{breedingMethodDbId}` | `/breedingmethods/{breedingMethodDbId}` | `GET` |
+| _node_.breedingmethods(_params_,...) | `/breedingmethods` | `/breedingmethods` | `GET` |
+| _node_.calls(_params_,...) | `/calls` | `/calls` | `GET` |
```

Comment:
you right, it looks confusing since retrieve different data. What about in different lines:
| _node_.calls(_params_,...) | `/calls` (retrieve server info) | | `GET` |
| _node_.serverinfo(_params_,...)||`/serverinfo` | `GET` |
| _node_.calls(_params_,...) | | `/calls` (retrieve genotyping calls) | `GET` |

Reply:
That'd work too!